pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -4,25 +4,141 @@
 package tpmmgr
 
 import (
+	"crypto/x509"
 	"flag"
 	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
 	"github.com/lf-edge/eve/pkg/pillar/agentlog"
 	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"time"
 )
 
 const (
+	//TpmPubKeyName is the file to store TPM public key file
+	TpmPubKeyName = "/config/tpm.eccpubk.der"
+
+	//TpmDevicePath is the TPM device file path
 	TpmDevicePath = "/dev/tpm0"
+
+	//TpmEnabledFile is the file to indicate if TPM is being used by SW
+	TpmEnabledFile = "/persist/config/tpm_in_use"
+
+	//TpmDeviceCertHdl is the well known TPM permanent handle for device key
+	TpmDeviceCertHdl tpmutil.Handle = 0x817FFFFF
+
+	defaultPassword = "\x01\x02\x03\x04"
+	emptyPassword   = ""
```

Comment:
I'm really uncomfortable with static passwords like these, unless there's a really good explanation.

Reply:
This is TPM owner password, to access TPM.   At the most one can delete the TPM secrets(not copy) with this password. To use this password, one has to gain root access to the machine. If he does, we have bigger problems anyway. Having said that, please share if you know of a better way, I will be happy to incorporate that.

Reply:
I suggest you generate it randomly and store on the box. That way, at least, the password is box-specific and not global in our code.

Reply:
well, this creates another set of problems. what if we lose the randomly generated password?

Reply:
@rvs please let me know your comments on this, as we need to make commit this code in.

Reply:
Addressed.  No static passwords now. Device generates a random password and uses that.